tests/integration/kubernetes/gha-run.sh

```diff
@@ -585,7 +585,10 @@ function main() {
 
 	# Auto-generate policy on some Host types, if the caller didn't specify an AUTO_GENERATE_POLICY value.
 	if [[ -z "${AUTO_GENERATE_POLICY}" ]]; then
-		if [[ "${KATA_HOST_OS}" = "cbl-mariner" ]]; then
+		if [[ "${KATA_HOST_OS}" = "cbl-mariner" || \
+			  "${KATA_HYPERVISOR}" = "qemu-snp" || \
```

Comment:
mine as well remove this line https://github.com/kata-containers/kata-containers/blob/main/.github/workflows/run-kata-coco-tests.yaml#L68 in this commit?